components/CompanionDelegate.tsx

```
@@ -0,0 +1,362 @@
+      }
+    },
+    [processPushNotifications, showClipboardAlert, wallets],
+  );
+
+
+
+  
+  const onNotificationReceived = useCallback(
+    async (notification: { data: { data: any } }) => {
+      const payload: any = Object.assign({}, notification, notification.data);
+      if (notification.data && notification.data.data) Object.assign(payload, notification.data.data);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)